community-plugins.json

```diff
@@ -849,5 +849,13 @@
         "author": "Joost Plattel",
         "repo": "jplattel/open-note-to-window-title",
         "branch": "main"
+    },
+    {
+        "id": "obsidian-plugin-todo",
+        "name": "Obsidian TODO",

```

Comment:
I think this is the `name` that gets searched/displayed rather than the `name` in the manifest, if you want to change it here too.

Reply:
Check! Updated it, thanks!